extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugInterface.java

```diff
@@ -34,54 +34,69 @@
 import java.util.Map;
 import java.util.Map.Entry;
 import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumInfoResponse;
+import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumsResponse;
 import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugResponse;
 import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugUserResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumsResponse;
+import org.dataportabilityproject.types.transfer.auth.AppCredentials;
+import org.dataportabilityproject.types.transfer.auth.TokenSecretAuthData;
+import org.scribe.builder.ServiceBuilder;
+import org.scribe.model.OAuthRequest;
+import org.scribe.model.Response;
+import org.scribe.model.Token;
+import org.scribe.model.Verb;
+import org.scribe.oauth.OAuthService;
 
 public class SmugMugInterface {
 
   // TODO(olsona): is this the mapper we want to use?
```

Comment:
Let's inject a mapper passed through from the TransferExtension here

Reply:
Done.